### PR TITLE
Use  instead of

### PR DESCRIPTION
**Pull request by LEDfan**
@tanghus This is needed to solve errors on PHP 5.3'. (I know I shouldn't use php5.3 but it's default on centos.....)

### CODE REVIEW

**Comment by LukasReschke** [comment]
Hehe - PHP 5.3 is still supported by ownCloud so don't feel ashamed ;-)
A good IDE can help finding such things while developing - try to get a PHPStorm license ;-)

File: lib/dispatcher.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ public function registerServices() {
 		});
 		$this->container->registerService('GroupController', function(IAppContainer $container) use($appName, $app) {
 			$request = $container->query('Request');
-			$tags = $this->server->getTagManager()->load('contact');
+			$tags = $container->server->getTagManager()->load('contact');
```
Comment (tanghus) [inline lib/dispatcher.php]: This should be

``` php
$tags = $container->getServer()->getTagManager()->load('contact');
```

File: lib/dispatcher.php [hunk 2]
```diff
@@ -89,7 +89,7 @@ public function registerServices() {
 		});
 		$this->container->registerService('ContactPhotoController', function(IAppContainer $container) use($appName, $app) {
 			$request = $container->query('Request');
-			$cache = $this->server->getCache();
+			$cache = $container->server->getCache();
```
Comment (tanghus) [inline lib/dispatcher.php]: Likewise here

File: lib/dispatcher.php [hunk 3]
```diff
@@ -98,7 +98,7 @@ public function registerServices() {
 		});
 		$this->container->registerService('ImportController', function(IAppContainer $container) use($appName, $app) {
 			$request = $container->query('Request');
-			$cache = $this->server->getCache();
+			$cache = $container->server->getCache();
```
Comment (tanghus) [inline lib/dispatcher.php]: And here

**Comment by tanghus** [comment]
Thanks for catching these issues @LEDfan 
Can I get you to correct the errors mentioned before I merge?

**Comment by LEDfan** [comment]
@tanghus the errors are fixed.

**Comment by tanghus** [comment]
That was fast :)

**Pull request merged**